_updates/2021-03-22-march.md

```diff
@@ -0,0 +1,40 @@
+# March 2021
+
+In the last month, we worked on two major tasks: settings page with a lot of new features and performance optimization of the admin ui and our packages. The long-awaited webhooks are finally available in sensenet. It opens up a lot of new opportunities for our customers.
+
+## admin-ui@1.18.0
+
+We added a new menu item called Settings to the sidebar navigation. ⚙️ Some previous items had been moved here like Localization, Setup (renamed to Configuration) and Personal settings (renamed to admin-ui customization). [#1159](https://github.com/SenseNet/sn-client/pull/1159)
+
+The Settings page also contains some new subpages: Stats 📊, Api and security 🔑, Webhooks 🔔.
+Stats page has a chart which shows the amount of storage space usage and the current utilization of your plan. It has 2 tables containing some technical details of your repository. [#1162](https://github.com/SenseNet/sn-client/pull/1162)
+
+Api and security is very useful if you want to build a new application over sensenet. You can manage your access tokens here which are needed for implementing authentication for your app. A similar page also available on profile.sensenet.com. [#1168](https://github.com/SenseNet/sn-client/pull/1168)
+
+The third new feature is webhooks, from now on you can subscribe to a lot of events in your repository and get notified by sending a request [#1172](https://github.com/SenseNet/sn-client/pull/1172). You can read more about webhooks on the [documentation site](https://docs.sensenet.com/concepts/webhooks)
+
+The admin-ui has a lot of features which means a lot of code. It can slow down the loading of our sites and destory the user experience. To prevent this we tried to minimize the size our JavaScript bundle loaded on the initial page load. We also removed some unused parts from our packages and made some code to load only on-demand. It produced excellent results:
```

Comment:
```suggestion
The admin-ui has a lot of features which means a lot of code. It can slow down the loading of our sites and destroy the user experience. To prevent this we tried to minimize the size our JavaScript bundle loaded on the initial page load. We also removed some unused parts from our packages and made some code to load only on-demand. It produced excellent results:
```